ksqldb-engine/src/main/java/io/confluent/ksql/function/udaf/offset/EarliestByOffset.java

```diff
@@ -45,7 +45,7 @@
 import org.apache.kafka.connect.data.Struct;
 
 @UdafDescription(
-    name = "EARLIEST_BY_OFFSET",
+    name = "EARLIEST_BY_OFFSET_OLD",
```

Comment:
This change is made in order to show that the new, generalized UDAF is being used.